deebotozmo/cli.py

```diff
@@ -0,0 +1,567 @@
+    finally:
+        await vacbot.after()
+
+
+@cli.command(
+    name="exportlivemap",
+    help='Get robot map and save it [filepath ex: "/folder/livemap.png"',
+)
+@click.argument("filepath", type=click.STRING, required=True)
+@click.option(
+    "--force-extension/--no-force-extension",
+    default=False,
+    help="Do not check if the file extension is valid.",
+)
+@click.pass_context
+@coro
+async def export_live_map(
+    ctx: click.Context, filepath: str, force_extension: bool
+) -> None:
+    """Click subcommand that returns the live map."""
+    if not force_extension and mimetypes.guess_type(filepath)[0] != "image/png":
+        logging.error("exportlivemap generates a png image.")
+        logging.error(
+            "either change your file extension to 'png' or pass '--force_extension'."
+        )
+        return
+
+    vacbot = CliUtil()
+    try:
+        await vacbot.before(ctx.obj["DEVICE"])
+
+        event = asyncio.Event()
+
+        async def on_map(_: MapEvent) -> None:
+            with open(filepath, "wb") as file:
+                file.write(base64.decodebytes(vacbot.bot.map.get_base64_map()))
+            event.set()
+
+        event.clear()
+        listener = vacbot.bot.events.map.subscribe(on_map)
+        await event.wait()
+        listener.unsubscribe()
+    finally:
+        await vacbot.after()
+
+
+@cli.command(name="getdevices", help="Get Devices")
+@click.option(
+    "--raw-data/--no-raw-data", default=False, help="Return raw data about all devices"
+)
+@coro
+async def get_devices(raw_data: bool) -> None:
+    """Click subcommand that returns all devices."""
+    vacbot = CliUtil()
+    try:
+        await vacbot.before()
+        if raw_data:
+            print(vacbot.devices)
+        else:
+            for idx, device in enumerate(vacbot.devices):
+                print(
+                    f"{idx + 1}) {device.nick} ({device.device_name}) ({device.did}) ({device.name})"
+                )
+    finally:
+        await vacbot.after()
+
+
+class CliUtil:
+    """CliUtil communicates with the bot to provide self.bot."""
+
+    def __init__(self) -> None:
+        """Init CliUtil."""
+        try:
+            config = read_config()
+        except configparser.MissingSectionHeaderError:
+            config_old = read_config_old()
+            config = configparser.ConfigParser()
+            config["account"] = {}
+            config["account"]["email"] = config_old["email"]
+            config["account"]["password_hash"] = config_old["password_hash"]
+            config["device"] = {}
+            config["device"]["device_id"] = config_old["device_id"]
+            config["device"]["continent"] = config_old["continent"]
+            config["device"]["country"] = config_old["country"]
+            config["device"]["verify_ssl"] = config_old["verify_ssl"]
+            write_config(config)
+            config = read_config()
+
+        device_id = config["device"]["device_id"]
```

Comment:
Makes no sense to add the config into two sections....
As all config values are relevant for the account and do not change on device base
Please use my first suggestion which the "global" section

Reply:
I thought it would be a good idea to use a config section for organization `[account]` section for email and password and another which is for deebotozmo.

Reply:
it looks like this now:
```
[account]
email = a@b.com
password_hash = b026324c6904b2a9cb4b88d6d61c81d1

[device]
device_id = 26ab0db90d72e28ad0ba1e22ee510510
continent = na
country = us
verify_ssl = True
```

Reply:
Please change it to
```
[DEFAULT]
email = a@b.com
password_hash = b026324c6904b2a9cb4b88d6d61c81d1
device_id = 26ab0db90d72e28ad0ba1e22ee510510
continent = na
country = us
verify_ssl = True
```

So it easy to add another account in the future, otherwise we need another refactoring, when we add support for multiple accounts. Also I think it would be great if they user can set a default device in the config, so he can skip the device on each request